client/my-sites/exporter/guided-transfer-options.jsx

```diff
@@ -18,7 +19,13 @@ export default React.createClass( {
 	},
 
 	purchaseGuidedTransfer() {
-		page( `/settings/export/${this.props.siteSlug}/guided` );
+		const { siteSlug } = this.props;
+		if ( config.isEnabled( 'manage/export/guided-transfer' ) ) {
+			page( `/settings/export/${ siteSlug }/guided` );
+		} else {
+			// Redirect to legacy guided transfer
+			window.location = `https://${ siteSlug }/wp-admin/paid-upgrades.php?product=40&view=purchase&source=export`;
```

Comment:
Also I'm not sure about the `product=40` part. If thats stored somewhere as a constant or anywhere else I think we could use it by name. If not then its probably not a big deal.